tron/utils/trontimespec.py

```diff
@@ -282,17 +297,7 @@ def get_first_day(month, year):
     # TODO: test
```

Comment:
TODONE?

Reply:
eh, no test actually calls handle_timezone, I'm going to leave it.